text/3771-demote-i686-pc-windows-gnu.md

```diff
@@ -0,0 +1,172 @@
+[explanation]: #explanation
+
+`i686-pc-windows-gnu` is now a [Tier 2 with Host Tools](https://doc.rust-lang.org/1.84.1/rustc/target-tier-policy.html#tier-2-target-policy) target instead of a [Tier 1 With Host Tools](https://doc.rust-lang.org/1.84.1/rustc/target-tier-policy.html#tier-1-target-policy) target.
+Official builds of the standard library and rustc **continue to be distributed** for this target, but it is no longer tested in CI.
+If necessary, further demotions (for example removing host tools) will not require RFCs, but go through a simpler [MCP](https://forge.rust-lang.org/compiler/mcp.html) instead.
+
+A blog post will be made to describe the change.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+By no longer doing automated testing for this target, this target will likely deteriorate more quickly than with continued automated testing.
+
+Additionally, this opens the door for further demotions in the future, like removing host tools, which could still be useful to some people.
+But such demotions will always have to be justified on their own.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+The maintenance requirement violation can be solved by multiple people stepping up to maintain this target. This has not happened so far.
+
+The popularity requirement could be fulfilled by more people using this target, but this does not seem possible, as 32-bit x86 as been on a decline for a long time, as new CPU models for this architecture are no longer being made.
+
+# Prior art
+[prior-art]: #prior-art
+
+This is the first time [since the Target Tier Policy was created](https://rust-lang.github.io/rfcs/2803-target-tier-policy.html) (note that this links to an old version, see [the rustc book](https://doc.rust-lang.org/1.84.1/rustc/target-tier-policy.html) for the latest version at the time of writing) that a Tier 1 target is being demoted.
+
+Before that, there has been the [demotion of `i686-apple-darwin` from Tier 1 to Tier 3 in 2019](https://github.com/rust-lang/rfcs/pull/2837).
+The reasoning there was mostly Apple's support being removed, which is not the case here.
+The measures in this RFC are much less drastic.
+
+The [promotion of `aarch64-apple-darwin` to Tier 1](https://github.com/rust-lang/rfcs/pull/3671) cited popularity as the major motivation, matching unpopularity as one of the motivations here.
+
+This is a continuation of [MCP 822](https://github.com/rust-lang/compiler-team/issues/822), which contains some additional details in the description and linked Zulip stream.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+None so far.
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+`x86_64-pc-windows-gnu` will remain a Tier 1 target after this RFC.
+While its popularity is more aligned with Tier 1, it suffers from the same lack of maintenance (but to a lesser degree) as its 32-bit cousin.
+It may be demoted as well in the future.
```

Comment:
Remark: at least the 64-bit windows-gnu is more accessible as a *host* target, unlike the 32-bit windows-gnu.

Reply:
if you want to cross-compile from linux to windows, the 64-bit windows-gnu (or gnullvm) target is the best way to do that, so I expect it to stay around a lot longer. we'd probably want to add an arm gnu target too at some point.

Reply:
I expect that we would have an aarch64-pc-windows-gnullvm or whatever target instead.

Reply:
This discussion is drifting away from the culprit, but I don't see value in ARM targets (32-bit).

AArch64 is already there as a cross compilation target (gnullvm only until GNU catches up), and once LLVM 20 goes live, I'll try to start MCP for promoting 64-bit gnullvm targets to hosts.

Reply:
> AArch64 is already there as a cross compilation target (gnullvm only until GNU catches up), and once LLVM 20 goes live, I'll try to start MCP for promoting 64-bit gnullvm targets to hosts.

nice! I had meant aarch64, but iirc windows doesn't call it that for some reason...